.buildkite/steps/parallel-steps/parallel-steps.yml

```diff
@@ -3,3 +3,5 @@ steps:
   - label: ":zap: Parallel Steps"
     command: ".buildkite/steps/parallel-steps/parallel-steps.sh"
     parallelism: 10
+  - label: ":mage: continue the adventure"
+    command: buildkite-agent pipeline upload ".buildkite/steps/ask/ask.yml" --log-level error
```

Comment:
What is instead of having this as a seperate step, we modified `.buildkite/steps/parallel-steps/parallel-steps.sh` to have something like this at the bottom of the file:

```sh
if [ $((BUILDKITE_PARALLEL_JOB+1)) -eq $BUILDKITE_PARALLEL_JOB_COUNT ]; then
  cd ../ask;
  pipeline_upload "ask.yml";
fi
```

Reply:
Nice!